### PR TITLE
Favicon coding

### PR DESCRIPTION
**Pull request by steveprutz**
Hello! This is my first PR, ever. I need the location of the favicon (I put the file in two places) reviewed... as well as the HTML header code.

### CODE REVIEW

File: favicon.png
Comment (steveprutz) [inline favicon.png:1]: Not certain where the "proper" place is for this favicon PNG to exist. Root or _Layouts?

Comment (DaneWeber) [reply favicon.png:1]: Favicon is pretty special, so the root is cool.

**Comment by DaneWeber** [comment]
I'm going to merge and see how this goes.

**Comment by DaneWeber** [comment]
Looks good.
image

**Pull request merged**